src/jquery.autofilter.js

```diff
@@ -63,15 +63,23 @@
         table._allColumnFilters = allColumnFilters;
     }
 
+    //Creating custom case insensitive expression for contains
+    jQuery.expr[':'].icontains = function(a, i, m) {
+        return jQuery(a).text().toUpperCase().indexOf(m[3].toUpperCase()) >= 0;
```

Comment:
Neat.  But what is the m[3] mean?

Reply:
m[3] is the actual input text value. I can add a comment in the code.